qemu/tests/seabios.py

```diff
@@ -62,18 +63,24 @@ def info_check():
             err_msg += " is %s" % get_output(vm.serial_console)
             raise error.TestFail(err_msg)
 
+    if not (boot_menu_hint and utils_misc.wait_for(boot_menu, timeout, 1)):
+        raise error.TestFail("Could not get boot menu message.")
```

Comment:
pls use test.fail() to replace raise error.TestFail

Reply:
@xiagao  updated